pages/common/ansible-vault.md

```diff
@@ -0,0 +1,28 @@
+# ansible-vault
+
+> Encrypts & decrypts values, data structures and files within Ansible projects.
+> More information: <https://docs.ansible.com/ansible/latest/user_guide/vault.html#id17>.
+
+- Create a new encrypted vault file with a prompt for a password:
+
+`ansible-vault create {{vault_file}}`
```

Comment:
Ideally this would be `path/to/vault_file`, but I think we want to keep it consistent across the entire page first - and with other pages second. In this case, we should probably leave it as-is because if we were to change the later examples, they would get rather long.